server/src/main/java/uk/ac/cam/cl/echo/extrusionfinder/server/preprocessor/ColorSpaceConversion.java

```diff
@@ -0,0 +1,180 @@
+package uk.ac.cam.echo.extrusionfinder.preprocessor;
+
+import java.lang.Math;
+
+import uk.ac.cam.cl.echo.extrusionfinder.server.imagedata.LabImageData;
+import uk.ac.cam.cl.echo.extrusionfinder.server.imagedata.RGBImageData;
+import uk.ac.cam.cl.echo.extrusionfinder.server.imagedata.XYZImageData;
+
+/**
+ * Collection of static colorspace conversion methods.
+ * Includes pairwise conversions between RGB, XYZ and L*a*b*.
+ */
+public class ColorSpaceConversion {
+
+    // The CIE XYZ tristimulus values of the reference white point for D6500
+    // wikipedia.org/wiki/Illuminant_D65#Definition
+    private static final double X_N = 0.95047;
+    private static final double Y_N = 1.00000;
+    private static final double Z_N = 1.08833;
+
+
+    // This composition is reccomended
+    /**
+     * Convert RGBImageData to LabImageData, preserving color
+     * <p>
+     * @param name Comment
+     * @throws Type Comment
+     * @return Comment
+     */
```

Comment:
What are this JavaDoc's param, throws, and return doing?
<p> not needed here.
Should terminate short description with full stop.
General lack of JavaDocs.